docs/core/compatibility/sdk/6.0/write-reference-assemblies-to-obj.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Breaking change: Write reference assemblies to IntermediateOutputPath"
+description: Learn about the breaking change in .NET 6 where reference assemblies are written to the IntermediateOutputPath by default.
+ms.date: 07/13/2021
+---
+# Write reference assemblies to intermediate output
+
+The .NET SDK now writes [reference assemblies](https://docs.microsoft.com/dotnet/standard/assembly/reference-assemblies) to the `IntermediateOutputPath` instead of the `OutDir` by default. This removes these build-time-only artifacts from your runtime-required outputs.
```

Comment:
```suggestion
The .NET SDK now writes [reference assemblies](../../../../standard/assembly/reference-assemblies.md) to the `IntermediateOutputPath` instead of the `OutDir` by default. This change removes these build-time-only artifacts from outputs that you require at run time.
```